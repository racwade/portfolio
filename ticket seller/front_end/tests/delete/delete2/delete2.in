login
test_buying
delete
simpleName
logout