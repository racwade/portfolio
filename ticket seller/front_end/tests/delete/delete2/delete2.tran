01 test_buy      BS 000009999