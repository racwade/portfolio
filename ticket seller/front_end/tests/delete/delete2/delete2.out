welcome to the ticket seller system
enter username:
logged in as test_buying
enter account username:
error: Only admin can deleted Accounts.
bye